robot/basestation/app.py

```diff
@@ -245,6 +247,36 @@ def capture_image():
 
     return jsonify(msg=msg)
 
+# This is called on if ffmpeg isn't already recording when the record toggle button is clicked
```

Comment:
I know we aren't utilizing this much but we should adhere to the PEP8 standards that we are trying to follow. The way to document stuff in python is like this https://www.python.org/dev/peps/pep-0008/#documentation-strings. You can learn more about them here : https://www.pythonforbeginners.com/basics/python-docstrings 

Additionally, I would not mention "when the record toggle button is clicked" because perhaps we would reuse that function from another source eventually.